apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/commons/AbstractSamhsaMatcherTest.java

```diff
@@ -453,11 +464,11 @@ public void shouldCorrectlyInvokeIsSamhsaCodingForProcedure() {
       // Invoke the captured lambdas to check they were the right ones
       icd9CallCaptor.getValue().test(mockCoding);
       icd10CallCaptor.getValue().test(mockCoding);
-      icd10CmCallCaptor.getValue().test(mockCoding);
+      icd10MedicareCallCaptor.getValue().test(mockCoding);
```

Comment:
Some places this seems to be replacing the icd10CM and some it seems to be in addition to it; which should it be? Documentation sounds like it should be an addition, so why are we not checking for both of these?

Reply:
After going through this code to add the ICD10 Medicare system check for procedures, I realize we don't need an ICD10 Medicare Diagnosis or ICD10CM Procedure SAMHSA check. So I decided to take out the ICD10 Procedure methods and checks I wrote.

Reply:
Gotcha. In the future it might be useful to include these kind of semi-related changes in the PR change notes area, just to clarify that some things arent directly related for context to reviewers.